packages/dev/core/src/Behaviors/Cameras/autoRotationBehavior.ts

```diff
@@ -23,6 +24,7 @@ export class AutoRotationBehavior implements Behavior<ArcRotateCamera> {
     private _idleRotationSpeed = 0.05;
     private _idleRotationWaitTime = 2000;
     private _idleRotationSpinupTime = 2000;
+    public _targetAlpha: Nullable<number> = null;

```

Comment:
public variable should not start with an underscore. you can change it to `targetAlpha`

Reply:
thanks. see the latest commit